src/Mapper/Mapper.ts

```diff
@@ -99,6 +80,14 @@ export class Mapper<T extends IModelData, R = T> implements IMapper<T, R> {
     return fromPairs(pairs) as any;
   }
 
+  /** Calls transform on the model and the model.original then creates a JSON patch to update the original to the updated */
+  public async transformPatch(model: IModel<T> | Partial<T> | any) {
+    const original = await this.transform(model.original);
```

Comment:
Where is `model.original` coming from? I don't see that property on `Model` or any of the underlying interfaces. 
EDIT: I see this is going to be introduced in IO-1907.